site/content@beta/platform-overview-new/platform-overview.md

```diff
@@ -0,0 +1,90 @@
+---
+weight: 40
+---
+
+# Extending the LiveChat UI
+
+<!-- [Extending user interfaces](https://developers.livechatinc.com/docs/building-apps-for-livechat/#extending-user-interfaces) -->
+
+One of the paths is to extend user interfaces. We provide developers with tools to modify the Agent App, as well as the Chat Widget interface. 
+
+## Agent App 
+
+![LiveChat Tag Master](livechat-tag-master-app.jpg)
+
+When integrating with the Agent App, developers can choose from a variety of options. They were designed to extend different locations of the Agent App making the whole application very flexible. To name them all, there are: 
+
+- Apps in the Details section (widgets)
+- Apps in the Customer Details section (Customer Detail widgets)
+- Chat Actions
+- Fullscreen apps displayed in the main menu
+- MessageBox
+
+Those are the possible integration interfaces that developers can work with. You can always check out our [Marketplace](https://www.livechatinc.com/marketplace/) to see what integrations are available for purchase.
+
+Each type of integration has its advantages, which in turn suggests use cases. The most common extensions are **apps in the Details section**, also known as widgets. They are quite versatile, which makes them usable in many cases. You could use them to display some statistics on your agents performance or to manage tags used by agents. Similar to **apps in Details** are the **apps in Customer Details**. There're perfect for displaying data needed at hand. 
+
+Speeding up the work can be easily achieved with **Chat Actions**, which are shortcuts to certain operations. Manual ticket creation takes a few steps, but with **Chat Actions** it's just a button click.
+
+![LiveChat Chat Actions](livechat-chat-actions.jpg)
+
+Web appplications can be displayed **fullscreen**, right in the Agent App. If you're planning on integrating your CRM with LiveChat, that might be the way to go.
+
+There's also another area where developers can integrate their apps- it's the Message Box. This interface can be used for extensions related strongly to messages, which are being sent.
+It could be used to send gifs (see [Integration with Giphy](https://developers.livechatinc.com/docs/building-apps-for-livechat/#messagebox)) or files from Dropbox.
```

Comment:
To może prowadzić do pytania o integrację z Giphy, która nie istnieje na dzień dzisiejszy nigdzie publicznie.